def check_exception(error):
    if error:
        raise Exception(error)